docs/2021-ga-release-blog-post.md

```diff
@@ -0,0 +1,80 @@
+# Introducing the new Azure SDK Management Libraries for Javascript
+
+We're excited to announce the new GA release for Javascript SDKs. These SDKs have several new features that help our customers to have a great experience in using the Azure Services. Some of the important SDKs that have been released are:
+
+1. [Compute](https://www.npmjs.com/package/@azure/arm-compute)
+2. [Network](https://www.npmjs.com/package/@azure/arm-network)
+3. [Storage](https://www.npmjs.com/package/@azure/arm-storage)
+4. [Sql](https://www.npmjs.com/package/@azure/arm-sql)
+5. [Resources](https://www.npmjs.com/package/@azure/arm-resources)
+6. [Policy](https://www.npmjs.com/package/@azure/arm-policy)
+7. [Managed applications](https://www.npmjs.com/package/@azure/arm-managedapplications)
+8. [Links](https://www.npmjs.com/package/@azure/arm-links)
+9. [Locks](https://www.npmjs.com/package/@azure/arm-locks)
+10. [Features](https://www.npmjs.com/package/@azure/arm-features)
```

Comment:
How about including the package names in this list? The text before the list talks about "libraries" but the list has text indicating services. Perhaps use something like `@azure/arm-compute` to manage Azure Compute`

Reply:
Done